src/Microsoft.DotNet.Wpf/src/System.Xaml/System/Xaml/Schema/XamlTypeInvoker.cs

```diff
@@ -42,47 +41,29 @@ public class XamlTypeInvoker
         [SecurityCritical]
         private ThreeValuedBool _isPublic;
 
-        // vvvvv---- Unused members.  Servicing policy is to retain these anyway.  -----vvvvv
```

Comment:
this might be fine, but i'd be a bit hesitant to remove these. @vatsan-madhavan or @rladuca do you have an idea what this servicing policy is and why these unused variables were kept in?